mmv1/products/dataproc/Batch.yaml

```
@@ -0,0 +1,504 @@
+    description: |
+      The time when the batch was created.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'runtimeInfo'
+    description: 'Runtime information about batch execution.'
+    output: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'outputUri'
+        description: |
+          A URI pointing to the location of the stdout and stderr of the workload.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'diagnosticOutputUri'
+        description: |
+          A URI pointing to the location of the diagnostics tarball.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'endpoints'
+        description: |
+          Map of remote access endpoints (such as web interfaces and APIs) to their URIs.
+        output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'approximateUsage'
+        description: |
+          Approximate workload resource usage, calculated when the workload completes.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'milliDcuSeconds'
+            description: |
+              DCU (Dataproc Compute Units) usage in (milliDCU x seconds)
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'shuffleStorageGbSeconds'
+            description: |
+              Shuffle storage usage in (GB x seconds)
+            output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'currentUsage'
+        description: |
+          Approximate workload resource usage, calculated when the workload completes.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'milliDcu'
+            description: |
+              Milli (one-thousandth) Dataproc Compute Units (DCUs).
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'shuffleStorageGb'
+            description: |
+              Shuffle Storage in gigabytes (GB).
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'milliDcuPremium'
+            description: |
+              Milli (one-thousandth) Dataproc Compute Units (DCUs) charged at premium tier
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'shuffleStorageGbPremium'
+            description: |
+              Shuffle Storage in gigabytes (GB) charged at premium tier.
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'snapshotTime'
+            description: |
+              The timestamp of the usage snapshot.
+            output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      The state of the batch.
+    output: true
+    values:
+      - :STATE_UNSPECIFIED
+      - :PENDING
+      - :RUNNING
+      - :CANCELLING
+      - :CANCELLED
+      - :SUCCEEDED
+      - :FAILED
```

Output-only enum fields aren't really meaningful, because the `values` field is unused. I'd recommend something like: 

```suggestion
  - !ruby/object:Api::Type::String
    name: 'state'
    description: |
      The state of the batch. For possible values, see the [API documentation](https://cloud.google.com/dataproc-serverless/docs/reference/rest/v1/projects.locations.batches#State).
    output: true
```